plugins/modules/win_domain_computer_info.py

```diff
@@ -0,0 +1,197 @@
+
+# Copyright: (c) 2020, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_domain_computer_info
+short_description: Gather active diretory information about a computer.
+requirements:
+  - This module requires Windows Server 2012 or Newer
+  - Requires ActiveDirectory module installed on system
+description:
+  - Gather information about 1 or more computers in Active Directory domain.
+options:
+  identity:
+    description:
+      - Specifies an Active Directory computer object by providing one of the following property values.
+      - A distinguished name, A GUID I(objectGUID), A security identifier I(objectSid), or A Security Accounts Manager account name I(sAMAccountName)
+    type: str
+  domain_server:
+    description:
+      - Specifies the Active Directory Domain Services instance to connect to.
+      - Can be in the form of an FQDN or NetBIOS name.
+      - If not specified then the value is based on the domain of the computer
+        running PowerShell.
+    type: str
+  domain_username:
+    description:
+      - The username to use when interacting with AD.
+      - If this is not set then the user Ansible used to log in with will be used instead when using CredSSP or Kerberos with credential delegation.
+    type: str
+  domain_password:
+    description:
+      - The password for domain targeted
+    type: str
+  properties:
+    description:
+      - list of propterties to pass to I(Get-AdComputer)
+      - case insensitive
+    type: list
+    elements: str
+    default: ['*']
+  excluded_properties:
+    description:
+      - list of properties to exclude from return
+      - case insensitive
+    type: list
+    elements: str
+    default: [ DeclaredConstructors, DeclaredMembers, nTSecurityDescriptor ]
+  search_scope:
+    description:
+      - Specify the scope of when searching for a computer.
+      - C(Base) will limit the search to the base object so the maximum number of objects returned is always one. This
+        will not search any objects inside a container..
+      - C(OneLevel) will search the current path and any immediate objects in that path.
+      - C(Subtree) will search the current path and all objects of that path recursively.
+    choices: [ Base, OneLevel, Subtree]
+    type: str
+  filter:
+    description:
+      - Specify filter to use with Get-Adcomputer
+    type: str
+  ldap_filter:
+    description:
+      - Specify ldap filter to use with Get-Adcomputer
+    type: str
+  search_base:
+    description:
+      - Specifies an Active Directory path to search under.
+    type: str
+seealso:
+  - module: ansible.windows.win_domain_object_info
+author:
+  - Larry Lane (@gamethis)
+'''
+
+EXAMPLES = r'''
+- name: Install ActiveDirectory Powershell extensions
+  ansible.windows.win_feature:
+    name: RSAT-AD-PowerShell
+    state: present
+
+- name: Install Active Directory Module
```

Comment:
Does this actually work? I thought it can only be installed from the Windows feature. I also cannot find any module called `ActiveDirectory` on the PowerShell Gallery.

Reply:
Yes it does... The active directory module is installed when you install RSAT-AD-PowerShell which is located on the server. it does not come from the PowerShell Gallery

Reply:
This step is using `win_psmodule` to install the module which is what I was referring to. Because it comes from `RSAT` then the step above is what is needed and this particular example isn't right AFAIK.